templates/email/order-cancel-failed.hypr

```diff
@@ -0,0 +1,22 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+<!--- Header --->
+<div class="mz-b2b-email-header">  
+  <div class="header-logo">
+    <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+  </div>  
+  <div class="header-text">
+    <div class="store-name">{{ siteContext.generalSettings.websiteName }}</div>
+  </div>
+</div>
+<br />
+<div>
+    <br /><br />
```

Comment:
Are 2 breaks really necessary? I don't see this being doubled up in other templates I just checked.

Reply:
Removed 'br' tag. Only for add some bottom space was using 'br' tag.